src/sentry/api/endpoints/issues/related_issues.py

```diff
@@ -33,6 +40,9 @@ def get(self, request: Request, group: Group) -> Response:
         :pparam Request request: the request object
         :pparam Group group: the group object
         """
+        serializer = RequestSerializer(data=request.query_params)
+        if not serializer.is_valid():
```

Comment:
@asottile-sentry thank you! I was doing it all wrong. I had thought it raises an error but it doesn't.